### DISCUSSION TITLE
Celery worker's memory keeps on increasing

### DISCUSSION THREAD

**Discussion by shauryagoel** [apache/superset#25593] [Q&A / Help]
I installed superset 2.1.0 using Helm. Currently, I have one worker pod whose memory keeps increasing even when the worker is doing no work. I have to restart the pod to release the memory. Is there a reason why this is happening?

**Comment by ndrluis** [comment]
Hi @shauryagoel, I’m having this problem and haven’t been able to solve it. Were you able to solve it in some way?

**Reply by shauryagoel** [reply]
I migrated to superset 3.0.1 and saw no unnecessary increase in memory so far.
